### ISSUE TITLE
[CoE Starter Kit - QUESTION] Issue: No Desktop Flows showing on Power Platform Admin View

### ISSUE THREAD

**Issue by harveyduan546** [microsoft/coe-starter-kit#3025]
### What is your question?

No more flows showing on Power Platform Admin View, please find the below screenshot
![image](https://user-images.githubusercontent.com/107934648/174813954-61629083-84f4-4b56-8c1c-fac9e9143d90.png)

And I found that Admin | Sync Template v3 (Desktop flows) runs normally as below screenshot
![image](https://user-images.githubusercontent.com/107934648/174814458-42bddeaf-8a0b-4da1-8ad0-87671b8bf167.png)

### What solution are you experiencing the issue with?

Core

### What app or flow are you having the issue with?

Power Platform Admin View

**Comment by Jenefer-Monroe** [comment]
What version of Core are you on? Do you have Desktop Flows in your tenant that you expect to see?

**Comment by Jenefer-Monroe** [comment]
Closing out as no further action for starter kit team

**Comment by harveyduan546** [comment]
Hello. can we re-open this one? since I just get the notification.

**Comment by Jenefer-Monroe** [comment]
Yes we can re-open, just so you know the process, I typically close out issues if I dont hear back in a week. 
Here are your current questions: 
What version of Core are you on? Do you have Desktop Flows in your tenant that you expect to see?

**Comment by Jenefer-Monroe** [comment]
We have some limitations documented on Desktop flows. If after reading these, you think you still have a unique issue we will need to find an instance to debug. 

Known limitations for Desktop Flows:
- [Desktop Flows in D4T environments](https://docs.microsoft.com/power-platform/guidance/coe/limitations#sync-flow-limitations-for-developer-and-microsoft-team-environments)
- [Large desktop flows](https://docs.microsoft.com/power-platform/guidance/coe/limitations#large-desktop-flows)

To find an instance to debug
1. Find a desktop flow outside of the above limitations
2. Note its envt
3. Go to your Power Platform Admin View app > Find the envt > Edit some field in the envt > Save
![image](https://user-images.githubusercontent.com/89584709/176202034-7a92d4a3-2504-4160-84f9-c901e12bf26a.png)

4. Go to the Desktop Flows sync flow run and select the timestamp of the flow that ran
![image](https://user-images.githubusercontent.com/89584709/176202292-29954731-9ef8-46fe-9b83-ecece072dc97.png)

This will allow us to debug with your example.

**Comment by harveyduan546** [comment]
Hi Jenefer

I read through two articles you provided.  I don’t my desktop flows fall into these two kinds. But I would like to point out there is no cloud flow to trigger my desktop flow.
I attached two screenshots according to your instructions.

***@***.***
***@***.***

Best Regards,
Harvey Duan

**Comment by harveyduan546** [comment]
One additional screenshot for your reference.
***@***.***


Best Regards,
Harvey Duan

From: Duan, Harvey
Sent: Wednesday, June 29, 2022 9:52 AM
To: microsoft/coe-starter-kit ***@***.***>; microsoft/coe-starter-kit ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [microsoft/coe-starter-kit] [CoE Starter Kit - QUESTION] Issue: No Desktop Flows showing on Power Platform Admin View (Issue #3025)

Hi Jenefer

I read through two articles you provided.  I don’t my desktop flows fall into these two kinds. But I would like to point out there is no cloud flow to trigger my desktop flow.
I attached two screenshots according to your instructions.

***@***.***
***@***.***

Best Regards,
Harvey Duan

**Comment by harveyduan546** [comment]
Here are some screenshots going with previous email.
![image](https://user-images.githubusercontent.com/107934648/176566932-8471e457-8562-474d-9c37-394b6af8e0ef.png)
![image](https://user-images.githubusercontent.com/107934648/176566951-65258b4d-b84c-4c78-9123-c6971cec85be.png)
![image](https://user-images.githubusercontent.com/107934648/176566963-a4fe33c4-6863-4730-b4d5-64a1b1c0e5dd.png)

**Comment by Jenefer-Monroe** [comment]
Sorry the instructions may not have been clear, let me try again

To find an instance to debug
1. Find a desktop flow outside of the above limitations (yours looks to be personal productivity)
2. Note its envt (you did the next steps on the CoE envt it sounds like, you need to do it on personal productivity, from step1)
3. Go to your Power Platform Admin View app > Find the envt > Edit some field in the envt > Save
![image](https://user-images.githubusercontent.com/89584709/176202034-7a92d4a3-2504-4160-84f9-c901e12bf26a.png)

4. Go to the Desktop Flows sync flow run and select the timestamp of the flow that ran
![image](https://user-images.githubusercontent.com/89584709/176202292-29954731-9ef8-46fe-9b83-ecece072dc97.png)

Once you have done this click on the timestamp to open the flow run

**Comment by harveyduan546** [comment]
I have two desktop flows on Personal Productivity envt, but I don't have cloud flow to trigger them so far. With regard to "trigger desktop flow inventory run", which envt should I make with it? The envt which the flow of Admin | Sync Template v3(Desktop flows) is on or the envt which desktop flows I created are in?

**Comment by Jenefer-Monroe** [comment]
you dont need a cloud flow to trigger it to inventory it, thats a limitation about flow runs not about the flows themselves. 
you need to trigger the envt which has the desktop flows (personal productivity from your picture)
![image](https://user-images.githubusercontent.com/89584709/176586281-e322841c-96ae-475e-9bde-38fc292f009e.png)

**Comment by harveyduan546** [comment]
please find the latest screenshot below.
![image](https://user-images.githubusercontent.com/107934648/176811763-4d257770-c458-4923-a30c-89b48ac0a613.png)
![image](https://user-images.githubusercontent.com/107934648/176812193-d79f188f-2652-4a45-b4bf-91080c182886.png)

**Comment by Jenefer-Monroe** [comment]
now you need to open that failure in order to investigate.

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation.
